testsuite/do_tests.sh.in

```diff
@@ -493,7 +493,8 @@ CODES_FAILURE=\
 
 run_test selftests/test_fail_or_die.sli "${CODES_SUCCESS}" "${CODES_SKIPPED}" "${CODES_FAILURE}"
 
-CODES_SUCCESS=' 3 Success'
+CODES_SUCCESS=' 3 Success,'\
+' 200 Skipped,'
```

Comment:
Skipping is not success. I think `CODES_SUCCESS` should remain unchanged. The `200` exit code from the script will then be registered and reported as a skipped test. `CODES_SKIPPED` is defined for the entire file around line 435.